examples/3d/frame_capture_cpu.rs

```diff
@@ -0,0 +1,117 @@
+use bevy::camera::{FrameCapture, FrameCapturePlugin};
+use bevy::core_pipeline::RenderTargetClearColors;
+use bevy::prelude::*;
+use bevy::render::camera::{CameraTypePlugin, RenderTarget};
+
+use bevy::render::render_resource::TextureFormat;
+use bevy::render::renderer::RenderDevice;
+
+#[derive(Component, Default)]
+pub struct CaptureCamera1;
```

Comment:
Is there a reason why this needs to be numbered?

Reply:
Just a left over from when I was testing using multiple cameras. I'll ditch the number.